views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -328,6 +358,18 @@ define([
                 this.$mediaStimulusContainer.removeClass('active');
             }
         },
+        /**
+         * Create countdown timer
+         */
+        initCountdown: function initCountdown() {
+            if (this.config.autoStart === true) {
+                var delayInSeconds = this.config.delayMinutes * 60 + this.config.delaySeconds;
```

Comment:
We don't need this variable because we only use it once.

Reply:
Indeed, the variable could be avoided. However, the bigger issue is the variable declaration is not following the good practices: when using the `var` keyword, the variable should be always be declared on top to reflect hoisting.

Reply:
Yes.